src/camera/SDL_camera.c

```diff
@@ -34,6 +34,9 @@ static const CameraBootStrap *const bootstrap[] = {
 #ifdef SDL_CAMERA_DRIVER_V4L2
     &V4L2_bootstrap,
 #endif
+#ifdef SDL_CAMERA_DRIVER_PIPEWIRE
+    &PIPEWIRECAMERA_bootstrap,
```

Comment:
How does this interact with the V4L2 driver? Should it be higher priority?

Reply:
ok, yes. Made it the first driver and removed the on-demand-only flag. seems to work and fall back correctly now.